README.md

```diff
@@ -573,41 +465,169 @@ For very custom setups, the plugin might be unable to find your scrollContainer
 },
 ```
 
+## variable-plugin
 
-## template-variable-plugin
+Editor plugin which provides node-specs and components which allow you to insert and edit different types of variables in a document. The plugin provides the following variable types, but this can be extended:
```

Comment:
did I miss something or is there no explanation what means "can be extended"?

Reply:
Indeed, I rephrased it a bit to make this clearer.